src/test/components/MarkerTooltipContents.test.js

```diff
@@ -232,14 +229,12 @@ describe('MarkerTooltipContents', function() {
             reason: 'CC_WAITING',
             slice: 1,
             pause: 5.23,
-            when: 17.5,
             budget: '11ms',
             initial_state: 'Initial',
             final_state: 'Final',
             major_gc_number: 1,
             page_faults: 1,
             start_timestamp: 17,
-            end_timestamp: 17,
```

Comment:
ping @PaulBone: I checked on a real profile and couldn't find `end_timestamp` and `when`. Can they be optionally present (in that case I'd add them in our types)?